### PR TITLE
template: remove unused stuff

### PR DESCRIPTION
**Pull request by MaskDuck**
**PLEASE PING ME FOR A REVIEW BEFORE YOU MAKE A CHANGE IN MY PATCH KTHX**

the removal of "how do you plan on using your domain" and "what is the purpose of these specified records" is intentional.

* we only ask for the purpose of these record when we are in doubt that it will be used for malicious stuff. sometimes the purpose of the records is very obvious e.g. maskduck.github.io is a website

* how do you plan on using these domain also serve like that.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,21 +1,14 @@
 ## Requirements
+Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.
+Please note that we reserve the rights not to accept any domain at our own discretion.
```
Comment (WilliamDavidHarrison) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion

Please note that we reserve the rights not to accept any domain at our own discretion.
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Add extra space.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (WilliamDavidHarrison) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: @MaskDuck

Comment (WilliamDavidHarrison) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: It makes it look better, so it's not a big blob of text no one will read. @MaskDuck

**Comment by s3mant** [comment]
wth are you guys doing 💀

**Comment by WilliamDavidHarrison** [comment]
> wth are you guys doing 💀

Duck doesn't like the changes from PRs #4430 & #4435

**Comment by s3mant** [comment]
I can see you two fighting to make shit prs of just changing few words in readmes💀

**Review by wdhdev — APPROVED** [review]

**Pull request merged**